eng/versioning/version_client.txt

```diff
@@ -6,7 +6,7 @@ com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-textanalytics;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-core;1.1.0;1.2.0
-com.azure:azure-core-amqp;1.0.0-beta.9;1.0.0-beta.9
+com.azure:azure-core-amqp;1.0.0-beta.9;1.0.1
```

Comment:
There is no beta.9 of azure-core-amqp https://search.maven.org/artifact/com.azure/azure-core-amqp only a beta.8, I'm not sure why both the dependency and current versions were set to the same version here but if you try and set 1.0.0-beta.9 as the dependency version then things should fail as there isn't one in Maven,.

Reply:
Agree This line should look like com.azure:azure-core-amqp;1.0.0-beta.8;1.0.1